### DISCUSSION TITLE
Data Importer - Salt Edge's Spectre API

### DISCUSSION THREAD

**Discussion by chris1284** [firefly-iii/firefly-iii#9273] [Firefly III] [answered]
Hi, 
in documentation is written:

`Limits
Note that Spectre does not offer the possibility of connecting to financial institutions via PSD2/Open Banking channels in the EU or the UK. These connections are limited to web scraping connections. Furthermore, there is a limit of 10 banks.

Since access to the Spectre API is limited to raw data only, you may have to do data cleanup yourself. You can use Firefly III [rules](https://docs.firefly-iii.org/how-to/firefly-iii/features/rules/) or build something yourself.

You can see if your bank is supported [on this page](https://www.saltedge.com/products/spectre/countries?channel=non_regulated).`

On the website my EU (German) Bank is listed like this

![image](https://github.com/user-attachments/assets/b07df702-2f27-4f3d-b852-67f097c9853f)

Is the doku outdated and now in EU/UK it possible to connect via PSD2/Open Banking?

Question 2: the importer with Salt Edge's Spectre API has only read right on bank accounts, correct?

Thx

**Comment by JC5** [comment]
It looks like Salt Edge got its PSD2 certificates and connections in order again. I may have to update the documentation, thanks for letting me know!

As for Q2: I believe they only ask for read access, yes. You can verify this in the dialog your bank presents you with.

**Comment by digitlength** [comment]
This is interesting @JC5 - I wonder if this means I could jump ship from GoCardless and get rid of the incoming tight rate limits. Not sure if the below is the page I should be looking at, but it seems that SaltEdge limits are per second, which is ideal...

https://docs.saltedge.com/v6/#rate-limits

**Comment by digitlength** [comment, marked as answer]
Sadly not - SaltEdge have said that the partnership with Firefly III does not cover integration of PSD2/Open Banking channels in the EU & UK.

We only get web scraping.
